content/python/concepts/random-module/terms/getrandbits/getrandbits.md

```diff
@@ -0,0 +1,50 @@
+---
+Title: .getrandbits()' 
+Description: 'Returns a pseudo-random integer with a specified number of bits.' 
+Subjects: 
+  - 'Computer Science'
+  - 'Data Science'
+Tags: # Please only use Tags in the tags.md file (https://github.com/Codecademy/docs/blob/main/documentation/tags.md). If that list feels insufficient, feel free to create a new Tag and add it to tags.md in your PR!
+  - 'Functions'
+  - 'Methods'
+  - 'Random'
+CatalogContent: 
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+---
+
+In the Python `random` module, the `.getrandbits()` method is used to generate a pseudo-random integer with a specified number of bits
```

Comment:
```suggestion
In the Python `random` module, the `.getrandbits()` method is used to generate a pseudo-random integer with a specified number of bits
```
Make sure that the first mention of the concept should always be in bold (**bold**).